src/ui/wxWidgets/AddEditPropSheetDlg.cpp

```diff
@@ -2046,6 +2206,59 @@ bool AddEditPropSheetDlg::IsGroupUsernameTitleCombinationUnique()
   return true;
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// TOTP Begin
+
+/// wxEVT_TIMER_EVENT event handler for ID_TIMER_TOTP_COUNTDOWN
+void AddEditPropSheetDlg::OnTotpCountdownTimer(wxTimerEvent& WXUNUSED(event))
+{
+  if (GetTotpItem() == nullptr) {
+    m_TotpTimer->Stop();
```

Comment:
Can we check `m_TotpTimer` for non-nullness, to make the code more robust?

Reply:
Changed